app/javascript/components/Download/RichContent.tsx

```diff
@@ -242,7 +250,19 @@ const FileEmbedGroupNodeView = ({ node }: NodeViewProps) => {
       if (file) files.push(file);
     });
     return [files, downloadInfo.hasStreamable(fileIds)];
-  }, [node.content.childCount, downloadInfo]);
+  }, [node?.content.childCount, downloadInfo]);
+
+  return { downloadableFilesInFolder, hasStreamable };
```

Comment:
Think it would be slightly nicer to just return this object from the memo directly